src/main/java/com.gigssandbox/UserActivityManager.java

```diff
@@ -0,0 +1,139 @@
+package com.gigssandbox;
+
+import com.gigssandbox.IO.OutputHandler;
+import com.gigssandbox.IO.InputHandler;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+class UserActivityManager {
+    private InputHandler inputHandler;
+    private OutputHandler outputHandler;
+    private UserActivityService userActivityService;
+
+    private String username;
+
+    UserActivityManager(InputHandler inputHandler, OutputHandler outputHandler) {
+        this.inputHandler = inputHandler;
+        this.outputHandler = outputHandler;
+        this.userActivityService = new UserActivityService();
+    }
+
+    void startInteracting() {
+        processUserLoggingIn();
+
+        while (true) {
+            displayPreloaded("choose_action");
```

Comment:
not sure that we should print action list each time. Its better to create help command. When you work with shell cli it doesn't print you list of commands